### ISSUE TITLE
Possible small bug

### ISSUE THREAD

**Issue by ssokota** [deepmind/open_spiel#827]
It looks like the code here is optimizing the policy but clipping the gradients for the value head

https://github.com/deepmind/open_spiel/blob/550f2fba696428606023e31ccd97cbf24c4a5daa/open_spiel/python/pytorch/policy_gradient.py#L457-L464

**Comment by lanctot** [comment]
@Asugawara can I ask you to take a quick look?

**Comment by Asugawara** [comment]
Hi!
I compare TensorFlow and PyTorch implementation again.
Clipping the value head grad which is detached on `pg_class.loss` method is meaningless.
Thank you for finding the bug and apologies for that.

**Comment by lanctot** [comment]
No problem. This is just a typo.. right? So the fix should be easy.. could you submit a PR?

**Comment by Asugawara** [comment]
>This is just a typo.. right? So the fix should be easy

Yes!
Please confirm this PR @lanctot @ssokota !!
https://github.com/deepmind/open_spiel/pull/828

**Comment by lanctot** [comment]
Fixed by https://github.com/deepmind/open_spiel/commit/c90e92537ce731c25623b1e55e722aa7793c7969
